### ISSUE TITLE
Org-noter and helm-M-x issues

### ISSUE THREAD

**Issue by Chobbes** [weirdNox/org-noter#16]
GNU Emacs 25.3.1 (x86_64-pc-linux-gnu) of 2018-02-23

I'm running emacsclient to connect to an emacs daemon, and when I open org-noter I get a new frame as expected.

Running helm-M-x kills the org-noter window, which is not the desired behaviour --- I want it to pull up the helm buffer for selecting which emacs command I want to run. I think the problem has to do with how helm is splitting org-noter to make the candidates buffer, which I believe will temporarily close the document buffer, which causes org-noter to close. If I switch over to the document buffer instead I can run helm-M-x as normal and it closes the notes buffer temporarily --- everything works as expected.

This is causing extra issues as well. I'm running xmonad, and I put org-noter in a separate desktop than the original emacsclient frame. When I do this and run helm-M-x it kills the org-noter buffer. Then when I switch to the other desktop the emacsclient frame is frozen. Mashing C-g / ESC does nothing. The only thing that seems to unfreeze it is... Starting another emacsclient.

If org-noter is in the same desktop as the other emacsclient frame it doesn't freeze.

Is there a way to make org-noter *not* immediately exit when the document buffer is temporarily closed?

**Comment by Chobbes** [comment]
Additional piece of info:

Helm is enabled with

(helm-autoresize-mode 1)

**Comment by weirdNox** [comment]
Hey! Sorry for the delay, I have been busy with college..

I have also encountered this issue when running `org-edit-special`, so I was thinking about removing this verification... I made it because I previously thought other commands would be more careful when the windows were dedicated, but it seems I was wrong. 

It was a way of having more isolation and terminating the session automatically when the user killed the frame, but I'll think about a way to remove it. :)
